packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -519,10 +514,10 @@ class _ReorderableListViewState extends State<ReorderableListView> {
 // of the objects used to generate widgets.
 @optionalTypeArgs
 class _ReorderableListViewChildGlobalKey extends GlobalObjectKey {
-  const _ReorderableListViewChildGlobalKey(this.subKey, this.state) : super(subKey);
+  const _ReorderableListViewChildGlobalKey(this.subKey, this.context) : super(subKey);
```

Comment:
This is a little iffy. We tell people to not hold on to context objects like that.